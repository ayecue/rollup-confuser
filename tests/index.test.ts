import { buildMockProject } from './utils';

describe('confuse', () => {
  test('default', async () => {
    const target = await buildMockProject();
    const module = require(target);

    expect(module).toMatchSnapshot();
  });
});