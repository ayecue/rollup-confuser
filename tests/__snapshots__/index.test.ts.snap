// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`confuse default 1`] = `
{
  "bar": [Function],
  "bubbleSort": [Function],
  "foo": [Function],
  "isPrime": [Function],
  "notBeingUseful": [Function],
  "unnecessarilyLongFizzBuzz": [Function],
}
`;
